webrender/src/render_backend.rs

```diff
@@ -25,8 +25,13 @@ use api::CapturedDocument;
 use clip_scroll_tree::{SpatialNodeIndex, ClipScrollTree};
 #[cfg(feature = "debugger")]
 use debug_server;
+#[cfg(not(feature="gleam"))]
+use device::PersistentlyMappedBuffer;
 use frame_builder::{FrameBuilder, FrameBuilderConfig};
 use gpu_cache::GpuCache;
+#[cfg(not(feature="gleam"))]
+use gpu_cache::{GpuBlockData, GPU_CACHE_INITIAL_HEIGHT};
+use hal;
```

Comment:
this include should be under `cfg(not(feature="gleam"))`

Reply:
We use the `B: hal::Backend` generics for the OpenGL backend as well with `gfx-backend-empty` so we need this include for that case as well.